src/schemata/definitions/schemaVersion.yml

```diff
@@ -1,6 +1,7 @@
 ---
 title: Schema Version
-description: A valid Workflow schema version.
+description: |
+  A valid Workflow Schema version.
```

Comment:
Previously this would be a `span` element, not it's using our mapped custom Typography component like all other descriptions.